yarn.lock

```diff
@@ -2,810 +2,524 @@
 # yarn lockfile v1
```

Comment:
Could you please remove this file?

Reply:
by *remove*, you mean reset it to the version in HEAD ? Will do.

Reply:
I accidentally committed this file, could you help to `git rm yarn.lock` for me?

Reply:
aaah, ok sure :)

Reply:
however, if you don't want it committed, shouldn't it be added to the `.gitignore` ?